From cd5004e525611e4257f1cf868849f4156cd74364 Mon Sep 17 00:00:00 2001
From: Erico Nunes <ernunes@redhat.com>
Date: Mon, 8 Apr 2019 21:30:39 +0200
Subject: [PATCH] fwts: do not use -Werror

Warnings come and go with various compiler versions, so using -Werror is
prone to cause build failures with various compiler versions, especially
newer versions that introduce new warnings.

Remove use of -Werror.

Upstream Status: not applicable

Signed-off-by: Erico Nunes <ernunes@redhat.com>
---
 configure.ac              | 2 +-
 src/Makefile.am           | 2 +-
 src/lib/src/Makefile.am   | 2 +-
 src/utilities/Makefile.am | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index f40c3678..0ff5025b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 	  AC_INIT([fwts],[0.1],[fwts-devel@lists.ubuntu.com])
-          AM_INIT_AUTOMAKE([-Wall -Werror foreign subdir-objects])
+          AM_INIT_AUTOMAKE([-Wall foreign subdir-objects])
           m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
 	  AC_CANONICAL_HOST
 	  AC_CONFIG_MACRO_DIR([m4])
diff --git a/src/Makefile.am b/src/Makefile.am
index f8066aff..6b979f80 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -12,7 +12,7 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir)/src/acpica/source/compiler	\
 	-I$(top_srcdir)/efi_runtime			\
 	-pthread `pkg-config --cflags glib-2.0 gio-2.0` \
-	-Wall -Werror -Wextra				\
+	-Wall -Wextra					\
 	-Wno-address-of-packed-member			\
 	-Wfloat-equal -Wmissing-declarations		\
 	-Wno-long-long -Wredundant-decls -Wshadow	\
diff --git a/src/lib/src/Makefile.am b/src/lib/src/Makefile.am
index 55c52b41..50efa97a 100644
--- a/src/lib/src/Makefile.am
+++ b/src/lib/src/Makefile.am
@@ -25,7 +25,7 @@ AM_CPPFLAGS = \
 	`pkg-config --silence-errors --cflags json-c`	\
 	`pkg-config --cflags glib-2.0 gio-2.0` 		\
 	-DDATAROOTDIR=\"$(datarootdir)\"		\
-	-Wall -Werror -Wextra				\
+	-Wall -Wextra					\
 	-Wno-address-of-packed-member
 
 pkglib_LTLIBRARIES = libfwts.la
diff --git a/src/utilities/Makefile.am b/src/utilities/Makefile.am
index a7d9b9c5..e1bf1411 100644
--- a/src/utilities/Makefile.am
+++ b/src/utilities/Makefile.am
@@ -16,7 +16,7 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #
 
-AM_CPPFLAGS = -Wall -Werror -Wextra -DDATAROOTDIR=\"$(datarootdir)\" 	\
+AM_CPPFLAGS = -Wall -Wextra -DDATAROOTDIR=\"$(datarootdir)\" 	\
 	-I$(srcdir)/../lib/include
 
 bin_PROGRAMS = kernelscan
-- 
2.31.1

